lab-gary/lib/greet.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports = exports = {};
+
+var strangerName = process.argv[2];
+
+exports.sayHi = function(name) {
+  if (arguments.length === 0) throw new Error('name not provided');
+  return `Hello ${name}!`;
+};
+
+exports.sayBye = function() {
+  console.log('See you later!');
+};
+
+exports.greetStranger = function() {
+  console.log(`Hi there, ${strangerName}!`);
```

Comment:
Much easier to pass the variable you made in here than the process.argv line you otherwise would have had to. It's much more human readable.